e2e/testdata/helm-charts/simple-2.5.3/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A simple test helm chart.
+name: simple
+version: 2.5.3
```

Comment:
I don't have permission to push this chart to the `kpt-config-sync-ci-release` project, so I will need someone to do that for me